src/components/MediaObject/MediaObject.tsx

```diff
@@ -41,6 +41,11 @@ export interface MediaObjectProps extends NestableBaseComponentProps {
    * content like buttons or other elements, pass those in as children instead of here.
    */
   extraContent?: React.ReactNode;
+
+  /**
+   * Limits text content to a single line, hiding additional text with an ellipsis.
+   */
+  ellipsis?: boolean;
```

Comment:
This will break existing implementations which expect the ellipsis by default. Most use cases will actually need the text wrapping, would it make more sense to change this to `allowTextWrap` and keep the default as ellipsed?

Reply:
the rest of the MediaObject sub-components could remain as-is, they are not exposed publicly

Reply:
@swese44 I chatted with @unindented about this briefly. `ellipsis` is being used already in other components. We felt that consistency with those components was more important.

Do you think we should go with allowTextWrap instead of updating the existing usages to include `ellipsis={true}`?

Reply:
Yeah, I feel that, if `Block` and `Text` already have an attribute `ellipsis`, and it is off by default, then adding an opposite attribute here would make things harder to understand.

Reply:
Block has `ellipsis`, Text has `maxWidth` which implicitly adds the ellipsis.

somebody deleted the UI Domain tenants from the group sidebar, but we had tenants like "provide smart defaults" and "make it hard to do the wrong thing, easy to do the right thing". if we default ellipsis to false here than most implementations will need to specify `ellipsis={true}`. we could default MediaObject's ellipsis to true, but that's the opposite of Block's default and might not be intuitive.

i'm ok with either prop name but i prefer `allowTextWrap`, and MediaObject should ellipse text by default

Reply:
if you guys feel strongly then we can go with the default value being false. if so we should publish a major version and update all existing instances in YamJS with `ellipsis={true}`

Reply:
Understood. My only worry then is referring to the same concept with three different names: `ellipsis`, `maxWidth` and `allowTextWrap`.

Reply:
ya it's a tradeoff. i still prefer defaulting to the main use case of ellipsed text and using `allowTextWrap` as the override. if you guys prefer defaulting to wrapped text with `ellipsis` then be sure to follow this up with a major version bump 5.0.0 and update instances in YamJS before anybody else bumps us to 5.x

Reply:
Okay, it sounds like the most common use case should use truncation via ellipsis, so I'll diverge from the `ellipsis` prop here. `allowTextWrap` incoming shortly!

Reply:
Can we rename `ellipsis` to something that has `textWrap` in the name then? `disallowTextWrap`? I still think naming the same concept with three different names in three components is a Bad Idea™.